package.json

```diff
@@ -12,7 +12,7 @@
       "require": "./dist/webxr.umd.cjs"
     }
   },
-  "version": "0.4.0",
+  "version": "0.6.0",
```

Comment:
What happened to v0.5.0?

Reply:
0.5.0 was prepped but never fully released since I've been quite busy on other projects.